doc/maintaining/installing/install-from-source.rst

```diff
@@ -21,7 +20,11 @@ work on CKAN.
 --------------------------------
 
 If you're using a Debian-based operating system (such as Ubuntu) install the
-required packages with this command for Ubuntu 16.04::
+required packages with this command for Ubuntu 18.04::
```

Comment:
AFAICT the list of packages is the same in both right? so can we simplify it a bit and just say

> install required packages with this command for Ubuntu 16.04 and 18.04::

Reply:
I agree. It looks much clearer this way.